pages/osx/archey.md

```diff
@@ -0,0 +1,19 @@
+# archey
+
+> Simple tool for stylishly displaying system information.
+
+- Show system information in stylish output:
```

Comment:
Hehe, no, it'd probably make sense to remove that "in stylish output" bit from the OS X one too.